scripts/publish-storage-account.sh

```diff
@@ -11,14 +13,16 @@ FILES="$(find sdk -name $BINARY_FILE_FILTER)"
 for file in $FILES
 do
     echo "Found binary" $file
-    cp $file $BITRISE_DEPLOY_DIR
+    cp $file $BUILD_ARTIFACTSTAGINGDIRECTORY
 done
 
 # Zip them
-cd $BITRISE_DEPLOY_DIR
+cd $BUILD_ARTIFACTSTAGINGDIRECTORY
 mkdir $ARCHIVE
 cp $BINARY_FILE_FILTER $ARCHIVE
 zip -r $ZIP_FILE $ARCHIVE
+cd -
 
 # Upload file
-$1azure storage blob upload $ZIP_FILE sdk
+AZURE_STORAGE_ACCESS_KEY=$AZURE_STORAGE_ACCESS_KEY \
```

Comment:
What is this line?

Reply:
expose environment variable just during that call

Reply:
Oh.. OK. Probably I need this in Apple SDK build. Currently it exports the environment variables.